2022_05_16/epitmenyado.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+
+
+def beolvasas():
+    df  = pd.read_csv("utca.txt", sep = " ", skiprows = 1, names = ["id", "s_name", "h_number", "tax_class", "area"])
+    return df
+
+def masodik_feladat(df):
```

Comment:
Ehhez a függvényhez is írj legyszi egy kicsi docstringet, és typehintet (a typehint amit a fenti példában is csináltam, hogy a függvény arg listánál megadod, hogy mi lesz az adott argumentum típusa)